prow/plugins/golint/golint.go

```diff
@@ -232,6 +251,9 @@ func handle(minimumConfidence float64, ghc githubClient, gc *git.Client, log *lo
 
```

Comment:
nit: I don't think we need to list the existing review comments if `len(problems) == 0` which is a possibility now.